src/services/message.service.ts

```
@@ -75,6 +115,36 @@ export class MessageService {
     return msg;
   }
 
+  private _failedToSend(message: IMessage, reason: string): boolean {
+    message.reply('Sorry the bot cannot send this message because:\n' + reason);
+    return false;
+  }
+
+  private _splitMessage(message: string, argBreakChar: string, numOfMiscChars: number): string[] {
+    const numOfFreeChars = Constants.MaxMessageLength - numOfMiscChars;
+    const breakChar = argBreakChar === '*' ? '(.|\n)' : argBreakChar;
+
+    const messageSplitRegExp: RegExp = new RegExp(
+      `(.|\n){1,${numOfFreeChars}}(${breakChar}|$)`,
+      'gm'
+    );
+
+    return message.match(messageSplitRegExp) || [];
```

shouldnt suppress this error